### DISCUSSION TITLE
sequence number

### DISCUSSION THREAD

**Discussion by Aaron-Wei** [Snipaste/feedback#2599] [Ideas]
![image](https://user-images.githubusercontent.com/57495183/236365033-48231657-8a32-4e90-b4d2-53f7b7ef235d.png)

Hope to add serial number, screen recording, long screenshot and other functions

**Comment by Geeyun-JY3** [comment]
- serial number needs Pro: https://github.com/Snipaste/feedback/issues/227#issuecomment-628125908
- screen recording: duplicate of #2130
- long screenshot: duplicate of #19
